### PR TITLE
Workflow logs are great again.  Closes #946

### LINKED ISSUE #946: Make workflows/{version}/{id}/logs great again
This is just a subset of the metadata REST API data... as a first pass, reuse as much as possible.  As a future path this could all be flavors of query (flexible filtering and column selection)

### PR DESCRIPTION
**Pull request by mcovarr**

### CODE REVIEW

File: engine/src/main/scala/cromwell/services/EngineMetadataServiceActor.scala [hunk 1]
```diff
@@ -91,6 +91,14 @@ case class EngineMetadataServiceActor(serviceConfig: Config, globalConfig: Confi
     }
   }
 
+  private def queryLogsAndRespond(id: WorkflowId): Unit = {
+    val replyTo = sender()
```
Comment (geoffjentry) [inline engine/src/main/scala/cromwell/services/EngineMetadataServiceActor.scala]: Mixing it up? :)

I like this better than `sndr` fwiw

Comment (mcovarr) [inline engine/src/main/scala/cromwell/services/EngineMetadataServiceActor.scala]: Yeah me too, I saw someone else doing this and decided to mend my ways.

Comment (geoffjentry) [inline engine/src/main/scala/cromwell/services/EngineMetadataServiceActor.scala]: I was never quite sure what you had against vowels

Comment (mcovarr) [inline engine/src/main/scala/cromwell/services/EngineMetadataServiceActor.scala]: maybe a good name for an Akka-based startup

**Comment by Horneth** [comment]
👍

**Comment by geoffjentry** [comment]
👍

**Pull request merged**